spec/tools/fix_auth/auth_model_spec.rb

```diff
@@ -25,9 +25,6 @@
   context "#authentications" do
     subject { FixAuth::FixAuthentication }
     let(:contenders) { subject.contenders.collect(&:name) }
-
-    # NOTE: these are not created unless you reference them
-    # if you want to always create them use let!(:var) {} instead
```

Comment:
These are actual comments ^

Reply:
Rather meaningless ones, though.

This spec is written rather awkwardly in the first place and should assign proper attributes on a per-test basis, not relying on that.

Reply:
I deleted it because I didn't think we should document RSpec in a spec file.  I would be surprised if anyone knew to look here to understand the difference between `let` and `let!`.

Reply:
True and true.  I guess the PR title is just slightly inaccurate.  :trollface: